community-membership.md

```diff
@@ -90,49 +92,47 @@ List of contributions:
-becoming a primary *reviewer* for the area that they are active in.
+becoming a primary *reviewer* for the subproject that they are active in.
 
 ## Reviewer
 
-Reviewers are able to review code for quality and correctness on some part of the project.
+Reviewers are able to review code for quality and correctness on some part of a subproject.
 They are knowledgeable about both the codebase and software engineering principles.
 
-**Defined by:** *reviewer* entry in an OWNERS file in the Kubernetes codebase.
+**Defined by:** *reviewers* entry in an OWNERS file in a repo owned by the Kubernetes project.
 
 Reviewer status is scoped to a part of the codebase.
 
 **Note:** Acceptance of code contributions requires at least one approver in addition to the assigned reviewers.
 
 ### Requirements
 
-The following apply to the part of codebase for which one would be a reviewer in an
-[OWNERS](contributors/devel/owners.md) file (for repos using the bot).
+The following apply to the part of codebase for which one would be a reviewer in an [OWNERS] file
+(for repos using the bot).
 
 - member for at least 3 months
 - Primary reviewer for at least 5 PRs to the codebase
 - Reviewed or merged at least 20 substantial PRs to the codebase
 - Knowledgeable about the codebase
-- Sponsored by an area approver
+- Sponsored by a subproject approver
   - With no objections from other approvers
   - Done through PR to update the OWNERS file
-- May either self-nominate, be nominated by an approver in this area, or be nominated by a robot
+- May either self-nominate, be nominated by an approver in this subproject, or be nominated by a robot
 
 ### Responsibilities and privileges
 
-The following apply to the part of codebase for which one would be a reviewer in an
-[OWNERS](contributors/devel/owners.md) file (for repos using the bot).
+The following apply to the part of codebase for which one would be a reviewer in an [OWNERS] file
+(for repos using the bot).
 
 - Tests are automatically run for PullRequests from members of the Kubernetes GitHub organization
 - Code reviewer status may be a precondition to accepting large code contributions
-- Responsible for project quality control via [code reviews](contributors/devel/collab.md)
+- Responsible for project quality control via [code reviews]
   - Focus on code quality and correctness, including testing and factoring
   - May also review for more holistic issues, but not a requirement
-- Expected to be responsive to review requests as per [community expectations](contributors/devel/community-expectations.md)
-- Assigned PRs to review related to area of expertise
-- Assigned test bugs related to area of expertise
-- Added to [`kubernetes-reviewers`](https://github.com/orgs/kubernetes/teams/kubernetes-reviewers)
+- Expected to be responsive to review requests as per [community expectations]
+- Assigned PRs to review related to subproject of expertise
```

Comment:
I think this was a search&replace snafu. Revert subproject to area here and on the next line.